src/context/OrderContext.js

```diff
@@ -0,0 +1,14 @@
+import { createContext } from "react";
+
+export default createContext({
+    isModeAdmin: false,
+    setIsModeAdmin: () => {},
+
+    // collapsed tab context
+    isCollapsed: false,
+    setIsCollapsed: () => {},
+
+    // Selected tab context 
+    currentSelectedTab: false,
```

Comment:
Ce ne serait pas plutôt une string ? 
```suggestion
    currentSelectedTab: "",
```